### PR TITLE
Rework to klog/v2

### PR DESCRIPTION
**Pull request by jsafrane**
**What type of PR is this?**
/kind cleanup
(?)

**What this PR does / why we need it**:

* Use klog2

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -7,14 +7,14 @@ require (
...
 	github.com/kubernetes-csi/csi-lib-utils v0.7.0
 	github.com/kubernetes-csi/csi-test/v3 v3.1.0
 	google.golang.org/grpc v1.28.0
-	k8s.io/api v0.18.0
-	k8s.io/apimachinery v0.18.0
-	k8s.io/client-go v0.18.0
+	k8s.io/api v0.19.0-rc.2
+	k8s.io/apimachinery v0.19.0-rc.2
+	k8s.io/client-go v0.19.0-rc.2
 	k8s.io/csi-translation-lib v0.18.0
```
Comment (humblec) [inline go.mod:18]: Cant csi-translation-lib upgraded too ? @jsafrane

File: go.mod [hunk 2]
```diff
@@ -7,14 +7,14 @@ require (
...
 	k8s.io/klog v1.0.0
```
[the remark this answers is not in the document]
Comment (jsafrane) [reply go.mod:19]: will do

File: go.mod (hunk 1 above)
Comment (jsafrane) [reply go.mod:18]: it's updated in #236
(maybe I could include it here too... I will check what breaks)

File: go.mod (hunk 2 above)
[the remark this answers is not in the document]
Comment (jsafrane) [reply go.mod]: done

File: go.mod (hunk 1 above)
Comment (jsafrane) [reply go.mod]: done

Comment (jsafrane) [reply go.mod]: #236 got fixed so I merged that. Reworking this PR to klog update only.

**Pull request merged**